src/content/en/fundamentals/codelabs/debugging-service-workers/index.md

```diff
@@ -3,7 +3,7 @@ book_path: /web/fundamentals/_book.yaml
 description: In this codelab, you'll learn how to debug a service worker using the new DevTools Application panel. You'll also learn how to simulate a Push notification to verify your subscription is properly setup.
 
 {# wf_auto_generated #}
```

Comment:
Whenever you see this tag, the source for the file comes from someplace else - in this case, it's from a Google Doc source, best to file a bug (and assign to me) and I'll get the right person to fix it in it's source.